content/docs/concepts/security/index.md

```diff
@@ -10,197 +10,186 @@ aliases:
+    link="./architecture.svg"
+    alt="Istio architecture."
+    caption="Istio Security Architecture"
+    >}}
 
-Istio tunnels service-to-service communication through the client side
-[Envoy](https://envoyproxy.github.io/envoy/) and the server side Envoy.
-Istio secures end-to-end communication via:
+In the following sections, we introduce the Istio security features in detail.
 
-- **Local TCP** connections between the service and Envoy.
+## Istio identity
 
-- **Mutual TLS** connections between proxies.
+Identity is a fundamental concept of any security infrastructure. At the beginning of a service-to-service communication,
+the two parties must exchange credentials with their identity information for mutual authentication purposes.
+On the client side, the server's identity is checked against the [secure naming](/docs/concepts/security/#secure-naming)
+information to see if it is an authorized runner of the service.
+On the server side, the server can determine what information the client can access based on the
+[authorization policies](/docs/concepts/security/#authorization-policy),
+audit who accessed what at what time, charge clients based on the services they used,
+and reject any clients who failed to pay their bill from accessing the services.
 
-- **Secure Naming**: during the handshake process, the client side Envoy checks
-  that the service account the server side certificate provided is allowed
-  to run the target service.
+In the Istio identity model, Istio uses the first-class service identity to be the identity of a service.
+This gives great flexibility and granularity to represent a human user, an individual service, or a group of services.
+On platforms that do not have such identity available,
+Istio can use other identities that can group service instances, such as service names.
 
-### Key management
+Istio service identities on different platforms:
 
-Istio supports services running on Kubernetes pods, virtual machines, and
-bare-metal machines. We use different key provisioning mechanisms for each
-scenario.
+- **Kubernetes**: Kubernetes service account
 
-For services running on Kubernetes pods, the per-cluster Citadel, acting as
-*Certificate Authority*, automates the key and certificate management process.
-Citadel mainly performs four critical operations:
+- **GKE/GCE**: may use GCP service account
 
-- **Generate** a [SPIFFE](https://spiffe.github.io/docs/svid) key and
-  certificate pair for each service account
+- **GCP**: GCP service account
 
-- **Distribute** a key and certificate pair to each pod according to the
-  service account
+- **AWS**: AWS IAM user/role account
 
-- **Rotate** keys and certificates periodically
+- **On-premises (non-Kubernetes)**: user account (custom service account), service name, istio service account, or GCP service account.
+  Custom service account refers to the existing service account alike identities managed by customer's Identity Directory.
```

Comment:
The custom service account refers to the existing service account just like the identities that the customer's Identity Directory manages.